.changeset/sixty-lions-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/shared': patch
```

Comment:
🙃 I think this is a `minor` version since we are extending our exposed API.

Reply:
No problem, i'll update the changeset